src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -91,8 +94,31 @@ All configuration files of ProActive Scheduler can be found under `PROACTIVE_HOM
 |User permissions|config/security.java.policy-server|<<_user_permissions>>
 |LDAP configuration|config/authentication/ldap.cfg|<<_ldap>>
 
+.5+^.^|Database
+|Scheduler configuration|config/scheduler/database.properties|<<Database configuration>>
+|Resource Manager configuration|config/rm/database.properties|<<Database configuration>>
+|Scheduling-api microservice|dist/war/scheduling-api/WEB-INF/classes/application.properties|<<Database configuration>>
+|Job-planner microservice|dist/war/job-planner/WEB-INF/classes/application.properties|<<Database configuration>>
+|Catalog microservice|dist/war/catalog/WEB-INF/classes/application.properties|<<Database configuration>>
+
 |===
 
+=== Database configuration
+
+Scheduler, Resource Mananger, and three microservices require to have direct access to database.
+Thus they have to be properly configured to have access to database.
+
+To configure Scheduler or Resource Mananger, one have to modify config/scheduler/database.properties
+and config/rm/database.properties respectively. It is necaessary to configure five following preperties:
```

Comment:
The following five properties must be configured:

Reply:
thanks